### PR TITLE
use manifest.json/actions.json args format for the config_panel.toml

### PR DESCRIPTION
**Pull request by Psycojoker**
## The problem

WIP work on this weekend, it depends on the #732 toml PR.

THIS REQUIRES A LITTLE BIT OF CHANGES FOR THE FORMAT OF THE CONFIG PANEL since now it's the manifest/actions format that is used.

The modifications are to this format in the [options] section, so, from head:

- bool -> boolean
- name -> ask

Here is a working (format) config https://gist.github.com/Psycojoker/c9ea335e596c4684b042812b7b0804c3

## Solution

...

## PR Status

...

## How to test

...

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

File: src/yunohost/app.py [hunk 1]
```diff
@@ -1672,42 +1675,43 @@ def app_config_apply(app, args):
...
-
     app_id, app_instance_nb = _parse_app_instance_name(app)
     env = {
         "YNH_APP_ID": app_id,
         "YNH_APP_INSTANCE_NAME": app,
         "YNH_APP_INSTANCE_NUMBER": str(app_instance_nb),
     }
     args = dict(urlparse.parse_qsl(args, keep_blank_values=True)) if args else {}
+    print(json.dumps(args, sort_keys=True, indent=4))
+    print("------------------------------------------")
```
Comment (alexAubin) [inline src/yunohost/app.py]: Randomly passing by but isn't this an old debug line ? (+ another json.dumps later)

Comment (Psycojoker) [reply src/yunohost/app.py]: Is it yes, the PR is WIP, I'll remove it once it's done

**Comment by Psycojoker** [comment]
Okay, that should be good, I've tested most of the things and that works for me ™

As a reference here is the script (and modify all the time) and config_panel I've used to test this feature. It's lazy as feuk but does its purpose.

@maniackcrudelis you can test it now if you want :)

I think that's all on the todo list before moving out of experimental? I'm not 100% sure but we probably start advertising it to apps dev if everything is fine and gather feedback once we feel ready.

I need to add the `.toml` format to the actions and the manifest too I think.

**Comment by Psycojoker** [comment]
Ah yeah, another important part is: how the hell are we going to do the CLI interface? (except if we spawn ncurse but that looks a bit overkill here) (while I really would love to put an urwid interface in yunohost that sucks for scripting)

**Pull request merged**